layout/_partial/Paradox-post_entry.ejs

```diff
@@ -1,4 +1,4 @@
-<div class="post_entry-module mdl-card mdl-cell mdl-cell--12-col fade out">
+<div class="post_entry-module mdl-card <% if(theme.card_elevation) { %>mdl-shadow--<%= theme.card_elevation %>dp<% } %> mdl-cell mdl-cell--12-col fade out">
```

Comment:
I would use `<% if(theme.card_elevation) { %><%= 'mdl-shadow--' + theme.card_elevation + 'dp' %><% } %>` for more readability